usng4j-api/src/main/java/org/codice/usng4j/CoordinatePrecision.java

```diff
@@ -64,16 +64,16 @@ public int getIntValue() {
     return this.precisionValue;
   }
 
-  public static CoordinatePrecision forEastNorth(int easting, int northing) {
-    int maxValue = Math.max(easting, northing);
+  public static CoordinatePrecision forEastNorth(String easting, String northing) {
+    int maxLength = Math.max(easting.length(), northing.length());
```

Comment:
Do we need to worry about trimming?

Reply:
We shouldn't since all of the easting/northing values should be trimmed already before calling `forEastNorth()` in the parse methods, but I added a trim just in case it gets missed in the future.